api/src/main/java/datawave/security/authorization/oauth/OAuthUserInfo.java

```diff
@@ -1,17 +1,12 @@
-package datawave.security.authorization;
+package datawave.security.authorization.oauth;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.google.common.collect.LinkedHashMultimap;
 import com.google.common.collect.Multimap;
-import com.google.common.collect.Multimaps;
 import datawave.security.authorization.DatawaveUser;
 import datawave.security.authorization.SubjectIssuerDNPair;
-import datawave.security.util.ProxiedEntityUtils;
 
 import java.util.Collection;
-import java.util.Collections;
-import java.util.LinkedHashSet;
 
 // Purpose of this class is to override the name field in DatawaveUser
 // to use the commonName instead of the subjectDN/issuerDN
```

Comment:
ignorable nit: I wonder if OAuthUser would be a better name for this because it doesn't extend DatawaveUserInfo, but DatawaveUser.

Reply:
This isn't a new class - this is available on main.  I just moved it to a different package and cleaned up some unused imports.  This is Bill's baby, so he might have an opinion about the naming.

Reply:
ok